ci/cloudbuild/builds/lib/features.sh

```diff
@@ -28,6 +28,7 @@ function features::always_build() {
     # These have hand-crafted code, therefore we always want to build them.
     bigtable
     experimental-bigquery_rest
+    experimental-opentelemetry_sdk
```

Comment:
I think I like `opentelemetry_exporters` better than `*_sdk`.

Reply:
The only thing stopping me is that the resource detector is not an exporter. What you suggest may still be better, though.

`opentelemetry_sdk` is named after "OpenTelemetry SDK". We would be perpetuating their fraught naming scheme.

Reply:
I'll take your suggestion and go with `opentelemetry_exporters` for now.

Reply:
Nevermind. I am merging and adding a discussion item to next meeting to sort out the names.

In the long run, we might just want to call the feature `opentelemetry`. The problem is that I am currently using `experimental-opentelemetry` to mean: compile the libraries with OpenTelemetry.